collector/zfs.go

```diff
@@ -0,0 +1,131 @@
+package collector
+
+// +build !nozfs
```

Comment:
this should also have a restriction to only run on linux and freebsd, as it won't compile on other platforms